h/db/__init__.py

```diff
@@ -63,6 +63,8 @@ def init(engine, base=Base, should_create=False, should_drop=False, authority=No
         # extension.
         engine.execute('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";')
         base.metadata.create_all(engine)
+        # Create sequences not used explicitly in a column here
+        engine.execute("CREATE SEQUENCE IF NOT EXISTS annotation_id_seq;")
```

Comment:
The sequence can't be added to the `Annotation` model somehow?

Reply:
I reckon once we start using it (eg as the server_default) this won't be necessary.